src/librustc_mir/const_eval.rs

```diff
@@ -527,6 +527,12 @@ pub fn error_to_const_error<'mir, 'tcx>(
     ConstEvalErr { error: error.kind, stacktrace, span: ecx.tcx.span }
 }
 
+pub fn undefined_behavior_error<'mir>() -> String {
```

Comment:
According to [the CI logs](https://dev.azure.com/rust-lang/rust/_build/results?buildId=3640), the `deny(unused_lifetimes)` lint is failing on the unused `'mir` lifetime. I strongly recommend building the compiler locally in order to catch these issues without having to wait for CI to pick up on your pull request: see the [chapter in the rustc development guide on how to build the compiler](https://rust-lang.github.io/rustc-guide/how-to-build-and-run.html).

Reply:
Thanks for your suggestion, now it enables to check in my local environment.